set.mm

```diff
@@ -99162,6 +99162,137 @@ a Cantor normal form (and injectivity, together with coherence
       JRS $.
   $}
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Well-Founded Recursion
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    $d A w y z $.  $d F w y z $.  $d G w y z $.  $d H w y z $.  $d R w y z $.
+    $( Functions defined by well-founded recursion are identical up to
+       relation, domain, and characteristic function.  General version of
```

Comment:
Is "characteristic function" standard ? I would have said "generating function", though I don't know the parlance in that domain.

Reply:
I can't find standard terminology here. I checked TZ, Levy, and Jech and none of them give a name to G. Generating function means something very specific in the analysis of recursive sequences though (see https://en.wikipedia.org/wiki/Generating_function). I'd prefer to avoid that name if possible.

Reply:
Yes, I know that meaning. Note that characteristic function also has another standard meaning !

Ok, keep it like that.